src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Vaccination/HomeVaccinationCellModel.swift

```diff
@@ -0,0 +1,42 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class HomeVaccinationCellModel {
+
+	// MARK: - Init
+
+	init(
+		healthCertifiedPerson: HealthCertifiedPerson,
+		onUpdate: @escaping () -> Void
+	) {
+		self.healthCertifiedPerson = healthCertifiedPerson
+		healthCertifiedPerson.$hasValidProofCertificate.sink { isValid in
+			self.isVerified = isValid
+			onUpdate()
+		}.store(in: &subscriptions)
+		healthCertifiedPerson.$proofCertificate.sink(receiveValue: { proofCertificate in
+			guard let proofCertificate = proofCertificate else {
+				self.vaccinatedPersonName = healthCertifiedPerson.healthCertificates.first?.name.fullName
+				onUpdate()
+				return
+			}
+			self.vaccinatedPersonName = proofCertificate.name.fullName
+			onUpdate()
```

Comment:
```suggestion
			self.vaccinatedPersonName = healthCertifiedPerson.fullName
			onUpdate()
```

I recently introduced a `fullName` property on the healthCertified person, you can also directly use that one.

Reply:
so i don't have to check if a proof certificate exists ?